### ISSUE TITLE
Panic during deserialization in `get_current_ruleset` function

### ISSUE THREAD

**Issue by selfuryon** [namib-project/nftables-rs#6]
Hello!
I tried to use `get_current_ruleset` function, but it panicked:

```console
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("data did not match any variant of untagged enum NfObject", line: 1, column: 1814)', src/firewall.rs:31:64
```

I use NixOS and my current ruleset (`nft -j list ruleset` with replaced IP addresses) is [here](https://gist.github.com/selfuryon/567f7581cc655938841a63df44af0adb).

**Comment by JKRhb** [comment]
Hi @selfuryon, thank you for reporting this issue! Could you try again with the latest crate version? :) With the fix provided in #8, the function should not panic anymore.

**Comment by jwhb** [comment]
@selfuryon: Thanks for providing your ruleset. There were indeed a some unimplemented pieces around sets and the XT statement.

Your ruleset can be interpreted after the changes that I committed to this branch: https://github.com/namib-project/nftables-rs/tree/anonymous-sets

Could you please confirm that this works for you?

**Comment by jwhb** [comment]
The fixes will be available as #9.
